docs/1-install-and-setup.md

```diff
@@ -155,22 +155,11 @@ And that's it! If it's totally *borked*, file an issue and let us know.
 
 #### 1. Cannot find namespace 'firebase'.
 
-If you run into this error while trying to invoke `ng serve`, open `src/tsconfig.json` and add the "types" array as follows:
-
-```json
-{
-  "compilerOptions": {
-    ...
-    "typeRoots": [
-      "../node_modules/@types"
-    ],
-    
-    // ADD THIS
-    "types": [
-      "firebase"
-    ]
-  }
-}
+If you run into this error while trying to invoke `ng serve`, you need to update to the correct types with the following commands (from the root of a new project):
+
```

Comment:
@mukesh51 
I've confirmed on multiple machines and a fresh Ubuntu VM. Also reported by others in my company. Is it possible this used to be an issue and therefore it's a version problem?